content/docs/install.md

```diff
@@ -1,58 +1,73 @@
 
 On newer versions of macOS, some headers are not installed by default. Open `macOS_SDK_headers_for_macOS_*.pkg` in `/Library/Developer/CommandLineTools/Packages/`.
+</details>
+
+<details>
+<summary>For GNU/Linux</summary>
+
+Make sure you have `llvm` and `clang` installed through your package managers.
+
+Make sure the LLVM binaries and the linker are added to your `$PATH` environmental variable.
+
+**Note**: The compiler currently relies on the `core` and `shared` library collection being relative to the compiler executable. Installing the compiler in the usual sense (to `/usr/local/bin` or similar) is therefore not as straight forward as you need to make sure the mentioned libraries are available. As a result, it is recommended to simply explicitly invoke the compiler with `/path/to/odin` in your preferred build system, or add `/path/to/odin` to `$PATH`.
+</details>
+
+## Build Odin
 
-Now navigate to the Odin directory in your terminal, use `make` or `./build.sh`, and you should have a newly-built, fresh Odin compiler!
+Now, it's time to build Odin and get started!
 
-### Updating the compiler
+<details>
+<summary>For Windows</summary>
 
-For a compiler that's in-development like Odin, things move fast. Make sure you keep your compiler up-to-date by running `git pull` and then rebuilding every now and then. (or, if you use releases, redownload and rebuild)
+The easiest way to build Odin is to use the X64 Visual Studio command prompt ([if you don't typically use it, here's how to find it](https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)). Navigate to the directory where you downloaded Odin. Run the `build.bat` file, and you should have a successfully built Odin compiler!
 
-## Setup
+To use Odin `link.exe` is required to be in the PATH of the callee as mentioned, this can either be achieved but calling Odin from the X64 Visual Studio command prompt or by calling the `vcvarsall.bat` (with x64 as an argument) script either in your shell or in your build script.
+</details>
 
-Odin only supports x86-64 at the moment (64-bit), relies on LLVM for code generation and an external linker.
+<details>
+<summary>For MacOS</summary>
 
-In addition, the following platform-specific steps are necessary:
+Navigate to the Odin directory in your terminal, use `make` or `./build.sh`, and you should have a newly-built, fresh Odin compiler! 
+</details>
 
-- Windows
-    * Have Visual Studio installed (MSVC 2010 or later, for the linker)
-    * Have a copy of `opt.exe` and `llc.exe` in `Odin/bin`. Pre-built Windows binaries can be found [here](https://github.com/odin-lang/Odin/releases/tag/llvm-windows) and *must* be explicitly copied
-    * Open a valid command prompt:
-        * **Basic:** run the `x64 Native Tools Command Prompt for VS2017` shortcut bundled with VS 2017, or
-        * **Advanced:** run `vcvarsall.bat x64` from a blank `cmd` session
```

Comment:
These steps are quite important and need to be made clear.a

Reply:
It was duplicated. It's confusing to have the same exact things twice.